apps/studio/tests/features/logs/LogsPreviewer.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { screen, waitFor } from '@testing-library/react'
+import dayjs from 'dayjs'
+import utc from 'dayjs/plugin/utc'
+import { beforeEach, expect, test, vi } from 'vitest'
+import { LogsTableName } from 'components/interfaces/Settings/Logs/Logs.constants'
+import LogsPreviewer from 'components/interfaces/Settings/Logs/LogsPreviewer'
+import { customRender, customRenderHook } from 'tests/lib/custom-render'
+import userEvent from '@testing-library/user-event'
+
+import useLogsPreview from 'hooks/analytics/useLogsPreview'
+import { LOGS_API_MOCKS } from './logs.mocks'
+import { addAPIMock } from 'tests/lib/msw'
+
+dayjs.extend(utc)
+
+vi.mock('common', async (importOriginal) => {
+  const actual = await importOriginal()
+  return {
+    useParams: vi.fn().mockReturnValue({}),
+    useIsLoggedIn: vi.fn(),
+    isBrowser: false,
+    LOCAL_STORAGE_KEYS: (actual as any).LOCAL_STORAGE_KEYS,
+    ...(actual as any),
+  }
+})
+
+vi.mock('lib/gotrue', async (importOriginal) => ({
```

Comment:
this feels like a pattern? 

vi.mock('lib/gotrue', async (importOriginal) => ({......

worth documenting in the readme?

Reply:
hmm i might be wrong here - maybe this could be in the global mock?